README.md

```diff
@@ -42,59 +42,37 @@ jobs:
         include:
         - language: go
           build-mode: autobuild
-        - language: java-kotlin
-          build-mode: none 
-        - language: javascript-typescript
-          build-mode: none
-        - language: python
-          build-mode: none
+          query: codeql/go-queries:AlertSuppression.ql        
```

Comment:
I would make the example for Java because that is one of the languages for which the language name in the codeql-action and the name of the codeql queries pack are slightly different. That should make it clear that those names could be different.
```yaml
- language: java-kotlin
  build-mode: none
  query: codeql/java-queries:AlertSuppression.ql 
```

Reply:
Makes sense, updated to java